osler/workup/forms.py

```diff
@@ -260,6 +261,26 @@ def clean(self):
 
         cleaned_data = super(WorkupForm, self).clean()
 
+        if 'pending' in self.data:
```

Comment:
I think you should be able to do if 'pending' in self.request.POST or something in the form_valid and then post to diff URLs that way since we try to put more logic in the views vs forms (See followup/views.py). If doing it this way makes it easier because of the required fields form clean though this is fine too!